strings/base_error.h

```diff
@@ -539,24 +539,11 @@ namespace winrt::impl
         return result;
     }
 
-    inline WINRT_IMPL_NOINLINE void check_cast_result(void* from, winrt::impl::slim_source_location const& sourceInformation = winrt::impl::slim_source_location::current())
+    inline WINRT_IMPL_NOINLINE void check_cast_result(hresult const result, winrt::impl::slim_source_location const& sourceInformation = winrt::impl::slim_source_location::current())
     {
-        if (!from)
+        if (result != 0)
         {
-            com_ptr<impl::IRestrictedErrorInfo> restrictedError;
-            if (WINRT_IMPL_GetRestrictedErrorInfo(restrictedError.put_void()) == 0)
```

Comment:
This removes the only calls to `WINRT_IMPL_GetRestrictedErrorInfo` and `WINRT_IMPL_SetRestrictedErrorInfo`. Remove them base_extern.h

Reply:
Done 👍